packages/apputils/src/inputdialog.ts

```diff
@@ -347,22 +397,49 @@ class InputNumberDialog extends InputDialogBase<number> {
 }
 
 /**
- * Widget body for input text dialog
+ * Base widget body for input text/password/email dialog
  */
-class InputTextDialog extends InputDialogBase<string> {
+class InputDialogTextualBase extends InputDialogBase<string> {
```

Comment:
The phrase `TextualBase` seems unusual here. Could we refer to this as `InputDialogTextBase` or `InputTextDialogBase`? The latter is more similar to the class name used before, and `Text` is used much more than `Textual` in front-end code, both ours, Lumino's, and other packages'.